[proxyAddresses] attribute is misspelled.

YuriySamorodov opened the pull request:
Line 828: [proxyAddresses] attribute is misspelled.
Changed from [proxyAddress] to [proxyAddresses]

azurecla commented:
Hi __@vidarmsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@vidarmsft This pull request contains merge conflicts.  Could you please resolve the merge conflict or close this pull request and submit a new one?  Thank you.

YuriySamorodov committed:
[proxyAddresses] attribute is misspelled.

Line 828: [proxyAddresses] attribute is misspelled.
Changed from [proxyAddress] to [proxyAddresses]
